app/src/main/res/layout/activity_intro3.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why are you using LinearLayout here ? Why can't we use a RelativeLayout. It works well if used with a Fragment for Maps.